packages/dataviews/src/types.ts

```diff
@@ -356,6 +356,11 @@ export interface ViewTable extends ViewBase {
 		 * The density of the view.
 		 */
 		density?: Density;
+
+		/**
+		 * Switch on/off hierarchical sorting.
+		 */
+		hierarchicalSort?: boolean;
```

Comment:
If this is a sort, should it be part of `view.sort` instead and layouts could decide to apply it or ignore it.